user-guide/Reference/faq/General_configuration.md

```diff
@@ -60,17 +60,29 @@ The time server will provide the correct time settings for all DMAs.
 
 To turn any computer into a time server, whether it serves as a DMA or not, do the following.
 
+> [!IMPORTANT]
```

Comment:
Instead of adding this as an "important" block, you should just combine it with the preceding paragraph. "To turn any computer using Windows Server 2008 R2, Windows 7, or later into a time server, ..."
"Important" blocks should only be used for things that really need to stand out because it could be a problem if the user doesn't take them into account, not for things that are just basic information.

Reply:
done